styleguide/docs/git-workflow.md

```diff
@@ -8,14 +8,12 @@ Emma Jane Westby has created wonderful videos and resources about using Git call
 
 ## Creating branches
 
-This project follows a git branching workflow using `feature/branchname`, `bugfix/branchname`, `hotfix/branchname`. When creating a new branch, it is helpful to base it off of `develop`. If you would like to include commits from a different branch that has not been merged, include those changes by checking out a new branch that is based on `develop`, and then use `git rebase origin/nameofbranchwithdesiredchanges`. This often works out better in the long run instead of 'chaining' feature branches off of each other directly.
+This project follows a git branching workflow using `feature/branchname`, `bugfix/branchname`, `hotfix/branchname`. When creating a new branch, it is helpful to base it off of `develop`. If you would like to include commits from a different branch that has not been merged, include those changes by checking out a new branch that is based on `develop`, and then use `git rebase origin/NameOfBranchWithDesiredChanges`. This often works out better in the long run instead of 'chaining' feature branches off of each other directly.
```

Comment:
@chelsiejohnston I thought this was ok to leave this given it's a very specific example and would be a non-destructive use of rebase, but I am also ok with you removing it and replacing it with a merge if you think it creates confusion.